nomos-services/storage/src/api/mod.rs

```diff
@@ -0,0 +1,42 @@
+use async_trait::async_trait;
+
+use crate::{
+    api::{
+        chain::{requests::ChainApiRequest, StorageChainApi},
+        da::{requests::DaApiRequest, StorageDaApi},
+    },
+    backends::StorageBackend,
+    StorageServiceError,
+};
+
+pub mod backend;
+pub mod chain;
+pub mod da;
+
+#[async_trait]
+pub trait StorageFunctions: StorageChainApi + StorageDaApi {}
+
+#[async_trait]
+pub trait StorageBackendApi: StorageFunctions
```

Comment:
One last question: why is `where Self: StorageChainApi + StorageDaApi` needed if `StorageBackendApi` is already a sub-trait of `StorageFunctions`, which is itself a subtrait of `StorageChainApi` and `StorageDaApi`? I think we can remove that, just for cleanliness.